server/routes/routerRequestHandlers/retrieveCuriousInPrisonCourses.test.ts

```diff
@@ -1,160 +1,34 @@
-import { NextFunction, Request, Response } from 'express'
-import { SessionData } from 'express-session'
+import { Request, Response } from 'express'
 import type { InPrisonCourseRecords } from 'viewModels'
-import { CuriousService } from '../../services'
+import CuriousService from '../../services/curiousService'
 import { aValidEnglishInPrisonCourse, aValidMathsInPrisonCourse } from '../../testsupport/inPrisonCourseTestDataBuilder'
 import retrieveCuriousInPrisonCourses from './retrieveCuriousInPrisonCourses'
 
+jest.mock('../../services/curiousService')
+
 describe('retrieveCuriousInPrisonCourses', () => {
-  const req = {
-    user: {} as Express.User,
-    session: {} as SessionData,
-    params: {} as Record<string, string>,
-    query: {} as Record<string, string>,
-    path: '',
-  }
+  const curiousService = new CuriousService(null, null, null) as jest.Mocked<CuriousService>
+  const requestHandler = retrieveCuriousInPrisonCourses(curiousService)
+
+  const prisonNumber = 'A1234GC'
+  const username = 'a-dps-user'
+
+  let req: Request
   const res = {
-    redirect: jest.fn(),
     locals: {} as Record<string, unknown>,
-  }
+  } as unknown as Response
   const next = jest.fn()
 
   beforeEach(() => {
     jest.resetAllMocks()
-    req.user = {} as Express.User
-    req.session = {} as SessionData
-    req.params = {} as Record<string, string>
-    req.query = {} as Record<string, string>
-    req.path = ''
-    res.locals = {} as Record<string, unknown>
-  })
-
-  const curiousService = {
-    getPrisonerInPrisonCourses: jest.fn(),
-  }
-
-  const requestHandler = retrieveCuriousInPrisonCourses(curiousService as unknown as CuriousService)
-
-  it('should retrieve prisoner In Prison Courses given In Prison Courses not already on res.locals', async () => {
```

Comment:
does this mean it was originally on res.locals and was moved to the session?

Reply:
actually this one was never in session, sorry my bad. Just checking but seems like these two tests did the same thing so you combined into one that just checks it is loaded?

Reply:
Yeah, there were actually pointless tests.
The implementation used `res.locals`, but because data on `res.locals` is not held/cached in the same way it would be on the session, these tests are effectively pointless! 🤣